<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LTNoteBook</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(30, 30, 30);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <item>
        <widget class="QPushButton" name="Open">
         <property name="minimumSize">
          <size>
           <width>31</width>
           <height>31</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/folder (1).png</normaloff>:/icon/folder (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>22</width>
           <height>22</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>0</number>
         </property>
         <property name="autoRepeatInterval">
          <number>2</number>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Save">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>31</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/disk.png</normaloff>
           <activeoff>:/C:/Users/zyx/Downloads/document.png</activeoff>
           <activeon>:/C:/Users/zyx/Downloads/document.png</activeon>:/icon/disk.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>0</number>
         </property>
         <property name="autoRepeatInterval">
          <number>200</number>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Delete">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/delete.png</normaloff>:/icon/delete.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>23</width>
           <height>23</height>
          </size>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Exit">
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/exit.png</normaloff>
           <normalon>:/C:/Users/zyx/Downloads/exit.png</normalon>
           <activeoff>:/C:/Users/zyx/Pictures/icon/exit.png</activeoff>
           <activeon>:/C:/Users/zyx/Downloads/exit.png</activeon>
           <selectedoff>:/C:/Users/zyx/Downloads/exit.png</selectedoff>:/icon/exit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="autoRepeat">
          <bool>false</bool>
         </property>
         <property name="autoRepeatDelay">
          <number>0</number>
         </property>
         <property name="autoRepeatInterval">
          <number>200</number>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Reset">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/left-arrow.png</normaloff>:/icon/left-arrow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Withdraw">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="res.qrc">
           <normaloff>:/icon/right-arrow.png</normaloff>:/icon/right-arrow.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>27</width>
           <height>27</height>
          </size>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="fileArrangement">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Microsoft YaHei</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>ClosedHandCursor</cursorShape>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LayoutDirection::LeftToRight</enum>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QComboBox::SizeAdjustPolicy::AdjustToMinimumContentsLengthWithIcon</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>KaiTi</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="tabChangesFocus">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(30, 30, 30);</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="Word">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>共 2 个字符</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFontComboBox" name="fontBox">
            <property name="minimumSize">
             <size>
              <width>148</width>
              <height>28</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Microsoft YaHei UI</family>
              <pointsize>10</pointsize>
              <italic>false</italic>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ClosedHandCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">/* 字体选择框样式 */
QFontComboBox {
    border: 2px solid #18e2cf;
    border-radius: 12px;
    background: #222831;
    color: white;
    padding: 6px 12px;
    font: 10pt 'Microsoft YaHei UI';
    min-height: 12px;
    min-width: 120px;
    selection-background-color: #18e2cf;
    selection-color: #222831;
}

QFontComboBox:hover {
    border: 2px solid #3de2cf;
    background: #2a3441;
}

QFontComboBox:focus {
    border: 2px solid #08FDD8;
    background: #2a3441;
    outline: none;
}

QFontComboBox:pressed {
    background: #1a2028;
    border: 2px solid #08FDD8;
}

QFontComboBox::drop-down {
    subcontrol-origin: padding;
    subcontrol-position: top right;
    width: 30px;
    border-left: 1px solid #18e2cf;
    border-top-right-radius: 10px;
    border-bottom-right-radius: 10px;
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #2a3441, stop:1 #222831);
}

QFontComboBox::drop-down:hover {
    background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #3a4451, stop:1 #2a3441);
    border-left: 1px solid #3de2cf;
}

QFontComboBox::down-arrow {
    image: none;
    width: 0;
    height: 0;
    border-left: 6px solid transparent;
    border-right: 6px solid transparent;
    border-top: 8px solid #18e2cf;
    margin: 0px 6px 0px 6px;
}

QFontComboBox::down-arrow:hover {
    border-top: 8px solid #3de2cf;
}

QFontComboBox::down-arrow:pressed {
    border-top: 8px solid #08FDD8;
}

QFontComboBox QAbstractItemView {
    border: 2px solid #18e2cf;
    border-radius: 8px;
    background: #232931;
    color: white;
    font: 10pt 'Microsoft YaHei UI';
    padding: 4px;
    selection-background-color: #18e2cf;
    selection-color: #222831;
    outline: none;
}

QFontComboBox QAbstractItemView::item {
    padding: 8px 12px;
    border: none;
    border-bottom: 1px solid #3a4451;
    background: transparent;
}

QFontComboBox QAbstractItemView::item:hover {
    background: #3a4451;
    color: #18e2cf;
}

QFontComboBox QAbstractItemView::item:selected {
    background: #18e2cf;
    color: #222831;
    font-weight: bold;
}

QFontComboBox QAbstractItemView::item:last {
    border-bottom: none;
}

/* 滚动条样式 */
QFontComboBox QAbstractItemView QScrollBar:vertical {
    background: transparent;
    width: 8px;
    margin: 0px;
    border-radius: 4px;
}

QFontComboBox QAbstractItemView QScrollBar::handle:vertical {
    background: #18e2cf;
    border-radius: 4px;
    min-height: 20px;
}

QFontComboBox QAbstractItemView QScrollBar::handle:vertical:hover {
    background: #3de2cf;
}

QFontComboBox QAbstractItemView QScrollBar::add-line:vertical,
QFontComboBox QAbstractItemView QScrollBar::sub-line:vertical {
    height: 0px;
}

QFontComboBox QAbstractItemView QScrollBar::add-page:vertical,
QFontComboBox QAbstractItemView QScrollBar::sub-page:vertical {
    background: none;
}</string>
            </property>
            <property name="iconSize">
             <size>
              <width>10</width>
              <height>10</height>
             </size>
            </property>
            <property name="currentFont">
             <font>
              <family>KaiTi</family>
              <pointsize>17</pointsize>
             </font>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnOpenSet">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="res.qrc">
              <normaloff>:/icon/setting.png</normaloff>:/icon/setting.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>22</width>
              <height>24</height>
             </size>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
            <property name="default">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QComboBox" name="fileCombobox">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>ClosedHandCursor</cursorShape>
            </property>
            <property name="maxVisibleItems">
             <number>8</number>
            </property>
            <property name="iconSize">
             <size>
              <width>18</width>
              <height>18</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>UTF-8</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UTF-16 LE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>UTF-16 BE</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ANSI</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Position">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>20</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>行1，列2</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
